certbot/main.py

```diff
@@ -782,6 +782,20 @@ def set_displayer(config):
         displayer = display_util.FileDisplay(sys.stdout)
     zope.component.provideUtility(displayer)
 
+def _post_logging_setup(config, plugins, cli_args):
+    "Perform any setup or configuration tasks that require a logger."
```

Comment:
We discussed this as a possibility, however, we still need to come up with that style and make sure there is an autoformatter we're happy with. In the meantime, I personally don't think that means that existing style/conventions go out the window.